### PR TITLE
Zio 1.0.0 rc21

### PR DESCRIPTION
**Pull request by regis-leray**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -27,24 +27,24 @@ inThisBuild(
 addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
 addCommandAlias("check", "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
 
-val zioVersion = "1.0.0-RC20"
+val zioVersion = "1.0.0-RC21"
```
Comment (mijicd) [inline build.sbt:30]: You can bump this one to use 21-1.

File: build.sbt [hunk 2]
```diff
@@ -27,24 +27,24 @@ inThisBuild(
...
-      "commons-net"              % "commons-net"              % "3.6",
-      "org.scala-lang.modules"   %% "scala-collection-compat" % "2.1.6",
-      "org.apache.logging.log4j" % "log4j-api"                % "2.13.1" % Test,
-      "org.apache.logging.log4j" % "log4j-core"               % "2.13.1" % Test,
-      "org.apache.logging.log4j" % "log4j-slf4j-impl"         % "2.13.1" % Test,
-      "dev.zio"                  %% "zio-test"                % zioVersion % Test,
-      "dev.zio"                  %% "zio-test-sbt"            % zioVersion % Test
+      "dev.zio"                 %% "zio"                     % zioVersion,
+      "dev.zio"                 %% "zio-streams"             % zioVersion,
+      "dev.zio"                 %% "zio-nio"                 % "1.0.0-RC7",
```
Comment (mijicd) [inline build.sbt:39]: Same here, RC8 is now available.

**Comment by regis-leray** [comment]
@mijicd thanks for your review, i applied the changes.
can you please verify my rights on this repo `zio-ftp` i dont have admin access, i have no issue with `zio-s3`

**Comment by mijicd** [comment]
@regis-leray Should be fixed now.

**Pull request merged**